packages/atomic/src/components/facets/facet-search/facet-search.tsx

```diff
@@ -184,15 +180,16 @@ export class FacetSearch {
   }
 
   private get searchResults() {
-    const showResults = this.component.showFacetSearchResults;
+    const showResults =
```

Comment:
I think, like @kaitlynkatz previously mentioned, that we need some kind of clear search in Headless, that would empty the query & the results without triggering search.
The problem I had here is "showFacetSearchResults" would show on focus but still display the older results (if I used the search before) meanwhile it was fetching the newer results.

I could work around this, by using more local variables or hiding on isLoading, but I rather add a clean method in Headless to allow this.

Reply:
Sounds good, we can add a `clear` method.

I just want to clarify the difference between two similar but slightly different UX interactions though:

1. If on blur text is not cleared

This is typically the case with the atomic search box. I would encourage people to hide the query suggest dropdown using CSS. When a user refocuses on the search box, the query suggestions will still be in state. There is no need to refetch them, saving a query and removing the lag to show the suggestions.

2. If on blur text is cleared

It makes sense to also clear the results. Providing a function that clears both text and results would definitely be helpful.

Reply:
(moved the comment here)

For the facet search, if we follow the JSUI ux work for it, we clear on blur (option 2)
But whether we do 1 or 2, the same issue still appears after selecting a result:
- input query e.g. `BBC `
- select result
- visually hide values and clear text (but don't retrigger a new search, which is useless)
- when focusing back on the search box, we visually show back the results, but until the new results are loaded we see still the `BBC` results, a headless clear would have previously emptied the values in the state.

Whether we do 1 or 2, we can't be sure that the results would still be the same, other parameters could have changed, so its safer to clear the result and redo a new search